packages/amplify_auth_cognito/example/ios/amplify_auth_cognito_tests/amplify_auth_cognito_tests.swift

```diff
@@ -0,0 +1,761 @@
+/*
+* Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+import XCTest
+import Amplify
+import AmplifyPlugins
+import AWSPluginsCore
+import AWSCore
+import AWSMobileClient
+@testable import amplify_auth_cognito
+
+// Test Data
+var _data: NSMutableDictionary = [:]
+var _args: Dictionary<String, Any> = [:]
+var _attributes: Dictionary<String, String> = [:]
+var _options: Dictionary<String, Any> = [:]
+var _username: String = "testuser"
+var _password: String = "mytestpassword"
+var _newPassword: String = "newPassword"
+var _oldPassword: String = "oldPassword"
+var _email: String = "test@test.com"
+var _phoneNumber: String = "+15555555555"
+var _confirmationCode: String = "confirmationCode"
+var _userId: String = "123"
+var _accessKey: String = "myAccessKey"
+var _secretKey: String = "mySecretKey"
+var _idToken: String = "myToken"
+var _accessToken: String = "myAccessToken"
+var _refreshToken: String = "myRefreshToken"
+
+class amplify_auth_cognito_tests: XCTestCase {
+    
+    var mockPlugin: SwiftAuthCognito = SwiftAuthCognito()
+    var mockCognito: MockCognito = MockCognito()
+    
+    override func setUpWithError() throws {
+        mockPlugin = SwiftAuthCognito.init(cognito: mockCognito)
+        _data = [:]
+        _args = ["data" : _data]
+        _attributes = [:]
+        _options = [:]
+    }
+
+    override func tearDownWithError() throws {
+        mockCognito.returnError = false;
+    }
+
+    func test_signUpSuccessEmail() {
+        _attributes = ["email" : _email]
+        _options = ["userAttributes": _attributes]
+        _data = [
+            "username": _username,
+            "password": _password,
+            "options": _options
+        ]
+        _args = ["data": _data]
+        let call = FlutterMethodCall( methodName: "signUp", arguments: _args )
+        mockPlugin.handle( call, result: {(result)->Void in
```

Comment:
Inconsistent formatting in this file `(call)` or `( call )`

Reply:
Fixed.